bin/ci-stack.ts

```diff
@@ -17,7 +17,11 @@ const defaultEnv: string = 'Dev';
 
 const ciConfigStack = new CIConfigStack(app, `OpenSearch-CI-Config-${defaultEnv}`, {});
 
-const ciStack = new CIStack(app, `OpenSearch-CI-${defaultEnv}`, {});
+const ciStack = new CIStack(app, `OpenSearch-CI-${defaultEnv}`, {
+  env: {
+    region: process.env.CDK_DEFAULT_REGION,
```

Comment:
If no input it will default to us-east-1?

Reply:
It will take the default region from the aws credentials, I think it defaults to `us-east-1`.